cli/runtime/plugin/doc.go

```diff
@@ -6,28 +6,35 @@ package plugin
 
 import (
 	"fmt"
+	"os"
 	"strings"
 
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/cobra/doc"
+
+	core "github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli/command/core"
```

Comment:
`cli/runtime` module should not have dependency on root level tanzu-framework module